test/lib/ufe/testSceneItem.py

```diff
@@ -42,7 +42,6 @@ def setUp(self):
         # Open top_layer.ma scene in testSamples
         mayaUtils.openTopLayerScene()
 
-    @unittest.skipIf(os.getenv('UFE_PREVIEW_VERSION_NUM', '0000') < '2020', 'testNodeTypes only available in UFE preview version 0.2.20 and greater')
```

Comment:
UFE v2 test only - no need for condition anymore. Strange that the test scene item if UFE v2 test only.